test/tasks/ods-build-go_test.go

```diff
@@ -199,6 +205,12 @@ func TestTaskODSBuildGo(t *testing.T) {
 					}
 				},
 				WantRunSuccess: true,
+				PostRunFunc: func(t *testing.T, ctxt *tasktesting.TaskRunContext) {
```

Comment:
why is there here this `PostRunFunc` and not in the others?

Reply:
If you unfold the others, you'll see that they are in fact also in a `PostRunFunc`. This specific test did not have a `PostRunFunc` so far as we only cared about success or failure. Now we want to check that the log message is not present as this was the only place where I could do it without creating a new test (not very clean, but saves test execution time ...)